[INFO][2023-11-13 12:03:46][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:03:46][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:03:46][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:03:46][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:03:46][victorinox.py:86] - use_tools=['meteo-weather', 'terminal', 'url-get', 'python'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-4-1106-preview', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 12:04:39][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:04:39][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:04:39][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:04:39][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:04:39][victorinox.py:86] - use_tools=['python', 'meteo-weather', 'url-get', 'terminal'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-4-1106-preview', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 12:06:27][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:06:27][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:06:27][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:06:27][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:06:27][victorinox.py:86] - use_tools=['terminal', 'url-get', 'meteo-weather', 'python'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-4-1106-preview', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 12:12:43][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:12:43][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:12:43][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:12:43][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:12:43][victorinox.py:86] - use_tools=['meteo-weather', 'url-get', 'terminal', 'python'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-4-1106-preview', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 12:14:43][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:14:43][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:14:43][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:14:43][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:14:43][victorinox.py:86] - use_tools=['meteo-weather', 'url-get', 'python', 'terminal'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-3.5-turbo', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 12:16:04][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:16:04][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:16:04][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:16:04][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:16:04][victorinox.py:86] - use_tools=['url-get', 'python', 'meteo-weather', 'terminal'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-3.5-turbo', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 12:24:34][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:24:34][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 12:24:34][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:24:34][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 12:24:34][victorinox.py:86] - use_tools=['python', 'meteo-weather', 'terminal', 'url-get'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-3.5-turbo', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 18:31:46][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:31:46][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:31:46][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 18:31:46][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 18:31:46][victorinox.py:86] - use_tools=['python', 'meteo-weather', 'terminal', 'url-get'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-4-1106-preview', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 18:43:05][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:43:05][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:43:05][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 18:43:05][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 18:43:05][victorinox.py:86] - use_tools=['terminal', 'python', 'meteo-weather', 'url-get'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-4-1106-preview', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 18:45:51][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:45:51][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:45:51][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 18:45:51][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 18:45:51][victorinox.py:86] - use_tools=['terminal', 'python', 'meteo-weather', 'url-get'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-4-1106-preview', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 18:47:10][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:47:10][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:47:10][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 18:47:10][chatgpt.py:186] - proxy no find, directly request to chatgpt instead
[INFO][2023-11-13 18:47:10][victorinox.py:86] - use_tools=['python', 'terminal', 'meteo-weather', 'url-get'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-4-1106-preview', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 18:48:04][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:48:04][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:48:04][chatgpt.py:184] - success use proxy: http://127.0.0.1:7890
[INFO][2023-11-13 18:48:04][victorinox.py:86] - use_tools=['url-get', 'terminal', 'python', 'meteo-weather'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': 'http://127.0.0.1:7890', 'request_timeout': 120, 'model_name': 'gpt-4-1106-preview', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-11-13 18:49:53][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:49:53][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-11-13 18:49:54][chatgpt.py:184] - success use proxy: http://127.0.0.1:7890
[INFO][2023-11-13 18:49:54][victorinox.py:86] - use_tools=['python', 'terminal', 'url-get', 'meteo-weather'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'deployment_id': '', 'proxy': 'http://127.0.0.1:7890', 'request_timeout': 120, 'model_name': 'gpt-4-1106-preview', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
